log_injection/exqlnet_chaoxing_sign/cx.py

```
@@ -0,0 +1,289 @@
+        return count
+
+    def server_chan_send(self, msg):
+        """server酱将消息推送至微信"""
+        if not self.sckey:
+            return
+        params = {
+            'text': '{}签到消息'.format(self.username),
+            'desp': msg,
+        }
+
+        requests.get('https://sc.ftqq.com/{}.send'.format(self.sckey), params=params)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)